### ISSUE TITLE
Noisy geometric surfaces when training "train_volume_renderer.py"

### ISSUE THREAD

**Issue by LiuXinqi12** [royorel/StyleSDF#28]
Hi,thanks very much for your exciting work. I encountered a problem when I retrained the model. When I use the "train_volume_renderer.py" with default parameters to train the HHFQ dataset, I find that there is always obvious noise on the reconstructed surface. Its output results do not generate smooth surfaces as the paper and the pre-training model. I found that the beta value at this time is around 0.0017. I tried to increase the weights of two geometrical related regular terms respectively(such as Eikonal Loss or Minimal Surface Loss), but they didn't seem to help significantly in smoothing the surface result. I would like to ask how you deal with this problem?  Looking forward to your reply, thanks again!

Current results：
![0171000](https://user-images.githubusercontent.com/47692145/212473266-9b48a81c-b422-4ec4-a0da-843675c97267.png)
![微信截图_20230114205459](https://user-images.githubusercontent.com/47692145/212473275-2dcce369-0fb6-4e0b-aab5-bb4222581169.png)

**Comment by royorel** [comment]
Hi @LiuXinqi12,

Can you please provide more details on how you extracted the surface? 
It looks like the volume renderer resolution is too low.

**Comment by LiuXinqi12** [comment]
Thanks very much for your reply and suggestions.  

During the retraining process, we directly modified the code in "train_volume_renderer. py" to predict the sdf using the “g_ema” generator, and then extracted the mesh. So, the rendering resolution is 64x64 by default, which is probably the main reason for noise. In contrast, the "generate_shapes_and_images.py" specially built a generator for surface extraction, which set the rendering resolution to 128X128. We will verify whether the surface noise is really caused by the rendering resolution. 

Thank you again for your valuable suggestions! ^ ^

**Comment by royorel** [comment]
It's not something I can directly verify, but the rendering resolution seems like the main issue from what I can see and the information you shared.
